templates/blog.html

```diff
@@ -1,5 +1,25 @@
 {% extends 'base.html' %}
 {% block head %}
+<!-- Open Graph Tags -->
+<meta property="og:type" content="article" />
+<meta property="og:title" content="{{ article.title|escape }}" />
+<meta property="og:site_name" content="{{ planet.name|escape }}" />
+<meta property="og:description" content="{{ article.content }}" />
```

Comment:
I am a little bit concerned that we put the whole content field into the description.

I will check the related implementation here to see if we can find a better way.

Reply:
I will inject a new variable for this purpose:

`article_summary`